### ISSUE TITLE
Is it possible that Python 3.7 is requried?

### ISSUE THREAD

**Issue by iwasherefirst2** [flathunters/flathunter#150]
Just reinstalling the application using Python 3.6 as recommended in the docs and now I see this:

> Warning: Your Pipfile requires python_version 3.7, but you are using 3.6.9 (/home/adam/.local/share/v/f/bin/python).
![image](https://user-images.githubusercontent.com/1765602/152638337-450adf3a-bb70-4e74-aa56-a042e458143f.png)

**Comment by codders** [comment]
Yeah. Would make sense. Looking at the git blame, it's been like this essentially since I took over the project. I've nobody else to blame :) I'll update it now. Thanks for the heads-up.

**Comment by iwasherefirst2** [comment]
Has been added to Readme
